scripts/shared/cloud-prepare.sh

```diff
@@ -27,18 +27,55 @@ function check_gateway_exists() {
     [[ $(kubectl get nodes -l "${GATEWAY_LABEL}" --no-headers | wc -l) -gt 0 ]]
 }
 
-function prepare_kind() {
-    read -r -a nodes <<< "${cluster_nodes[$cluster]}"
-    local node=${cluster}-${nodes[-1]}
-    kubectl label node "$node" "$GATEWAY_LABEL" --overwrite
+function check_and_apply_air_gap_env() {
```

Comment:
A better name would be `prepare_kind_node`, and you can keep the labeling logic inside

Reply:
@mkolesnik 
I created a separate function for the air gup config to not duplicate the same lines.

Reply:
Yes you duplicate the labeling call, which also needs to happen per node, hence my comment on the "before" code - https://github.com/submariner-io/shipyard/pull/944/files#r1002693431

Reply:
Fixed.